src/py-o365-smtp/config.toml

```diff
@@ -0,0 +1,4 @@
+# config.toml
+
+[graph]
+endpoint = "https://graph.microsoft.com/v1.0/me/messages"
```

Comment:
My plan was to use SMTP instead of Graph.  The reason being is that I've had a few requests for one SMTP destination in prod, and another non-o365 destination in dev/test.  Similar to the PODE server that @kpoineal built for RP.